scss/blocks/figma-component-props-table.scss

```diff
@@ -0,0 +1,120 @@
+:root {
+    --figma-component-props-table-background-color: rgba(244, 245, 247, 0.95);
+    --figma-component-props-table-badge-border-color: #ECEEF3;
+}
+
+table.props-table {
+    width: 100%;
+}
+
+table.props-table thead {
+    background-color: var(--figma-component-props-table-background-color);
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+    color: #606981;
+}
+
+table.props-table th {
+    border-top: none;
+    border-bottom: 1px solid #ECEEF3;
+}
+
+table.props-table th:first-child{
+    border-radius: 8px 0 0 0;
+}
+
+table.props-table th:last-child{
+    border-radius: 0 8px 0 0;
+}
+
+table.props-table td {
+    border-top: none;
+    border-bottom: 1px solid #ECEEF3;
+    padding: 12px;
+}
+
+table.props-table .props-table-value {
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 20px;
+    color: #14171F;
+    background-color: var(--figma-component-props-table-background-color);
+    padding: 2px 4px;
+    border-radius: 4px;
+    display: flex;
+    align-items: center;
+    gap: 4px;
+    width: fit-content;
+}
+
+table.props-table .props-table-name {
+    font-size: 14px;
+    font-weight: 500;
+    line-height: 20px;
+    color: #14171F;
+    display: flex;
+    gap: 8px;
+}
+
+.props-table-variant-badge {
+    color: #0F62FE;
+    background-color: #EDF5FF;
+    border: 1px solid var(--figma-component-props-table-badge-border-color);
+    border-radius: 4px;
+    padding: 2px 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+}
```

Comment:
Those seems to be repeated all over. They should have some common parent for the same styles, something like `badge`. It would also make sense to create separate template for this and just use the template for badge I think. But if it is used only here for now then I would leave it on you as optional

Reply:
[71d4484](https://github.com/Supernova-Studio/exporter-documentation/pull/166/commits/71d44841d7439094e36588bb37b4f482eaf8818d)